doc/under_sampling.rst

```diff
@@ -116,7 +122,8 @@ In addition, :class:`RandomUnderSampler` allows to sample heterogeneous data
   >>> print(y_resampled)
   [0 1]
 
-It would also work with pandas dataframe::
+:class:`RandomUnderSampler` can also take a pandas dataframe as input for
```

Comment:
Actually, all samplers support it. It might be better to just remove this statement and make sure that we have a section about it.

```suggestion
:class:`RandomUnderSampler` also supports pandas dataframes as input for
```